codecov.yaml

```diff
@@ -37,10 +37,9 @@ coverage:
         flags:
           - unit-tests
 
-
 flag_management:
   default_rules:
-    carryforward: true
+    carryforward: false
```

Comment:
**Suggestion:** Consider setting 'carryforward' to true to keep the coverage from the last successful commit when a file has no coverage data for the current commit. This can prevent sudden drops in coverage.
```suggestion
    flag_management:
      default_rules:
        carryforward: true
```